.github/workflows/rust.yml

```diff
@@ -86,7 +86,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ ubuntu-18.04, macos-11 ]
+        os: [ ubuntu-18.04 ]
```

Comment:
this doesn’t need to be a matrix anymore. not sure it’s worth changing

Reply:
oh I guess it’ll keep the job name stable if you leave it

Reply:
I would leave it even if we only intend to have one for now because I renamed the Helios job to match the form

Reply:
yeah i thought about this and went with "keep the naming stable"

plus this whole stanza is probably gonna get moved into buildomat anyway